packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -235,6 +235,16 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
 /// is a full-featured, material-design text input field with placeholder text,
 /// labels, and [Form] integration.
 ///
+/// ## Gesture Events Handling
+///
+/// This widget will provide rudimentary, platform-agnostic gesture event
+/// handling for user actions such as tapping, long pressing and scrolling when
```

Comment:
long-pressing, and

What's the "user action" for scrolling. Maybe "drag-scrolling"?

Reply:
For the second part, I don't think users will be confused